### PR TITLE
SDK-34: JAVA SDK: set a limit on minimum polling interval

### PR DESCRIPTION
**Pull request by abhisheksr8**
Setting minimum poll interval to 1 second.
If anything less than this is specified then it defaults to 1 second.

### CODE REVIEW

File: src/main/java/com/qubole/qds/sdk/java/client/ResultLatch.java [hunk 1]
```diff
@@ -70,7 +71,12 @@ public ResultLatch(QdsClient client, String queryId)
      */
     public void setPollSleep(long time, TimeUnit unit)
     {
-        pollMs.set(unit.toMillis(time));
+    	if(unit.toMillis(time)<MIN_POLL_INTERVAL_MS){
+    		pollMs.set(MIN_POLL_INTERVAL_MS);
+    	}
+    	else{
+    		pollMs.set(unit.toMillis(time));
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/client/ResultLatch.java]: Please follow existing indentation.

File: src/main/java/com/qubole/qds/sdk/java/client/ResultLatch.java [hunk 2]
```diff
@@ -70,7 +71,12 @@ public ResultLatch(QdsClient client, String queryId)
      */
     public void setPollSleep(long time, TimeUnit unit)
     {
-        pollMs.set(unit.toMillis(time));
+    	if(unit.toMillis(time)<MIN_POLL_INTERVAL_MS){
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/client/ResultLatch.java]: Need to add spaces after `if`, before and after `<` and after `)`.
Please follow existing code style.

File: src/main/java/com/qubole/qds/sdk/java/client/ResultLatch.java [hunk 3]
```diff
@@ -70,7 +71,12 @@ public ResultLatch(QdsClient client, String queryId)
...
+    		pollMs.set(MIN_POLL_INTERVAL_MS);
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/client/ResultLatch.java]: Add a log statement which warns the user that we have overridden the polling interval.

**Comment by abhisheksr8** [comment]
made indentation/space related changes

**Comment by mindprince** [comment]
Squash merged. 636189a

**Pull request closed without merging**